src/components/Video/Video.module.css

```diff
@@ -111,3 +111,12 @@
     object-fit: cover;
   }
 }
+
+.mediaBlock {
+  @media (--viewport-xs-sm-md-only) {
+    padding-top: 0;
+  }
+  @media (--viewport-lg) {
+    padding-top: 0;
```

Comment:
Like this 
```suggestion
  padding-top: 0;
```

Reply:
If using above without a media query, it will not be overwritten.
![Screen Shot 2021-11-09 at 9 31 44 am](https://user-images.githubusercontent.com/29698458/140828636-5322f198-a18b-4ea3-bd34-6409fb96e2aa.png)

Reply:
Hmmm 🤔 I'll leave this one to @kevin-ho87, he's much better at css than I am 😄

Reply:
😝 all good. Thanks 😌

Reply:
ah specificity issue. you could do something like
```suggestion
.mediaBlock.mediaBlock {
  padding-top: 0;
}
```

Reply:
That works, thanks kevin